### PR TITLE
eventpoll: resort toff back by idx

### PR DESCRIPTION
**Pull request by Snorch**
In dump_one_eventpoll we call find_tfd_bsearch with arguments
"e->tfd[i]->tfd, toff[i].off". By this we want to check if
"toff[i].off"-th inclusion of target "e->tfd[i]->tfd" in an eventpoll
coresponds to our fd "e->tfd[i]->tfd" in terms of kcmp.

But because of toff was sorted to calculate off, indexes in e->tfd and
toff does not address the same eventpoll target at this point.

Let's sort toff back to original state to make indexes consistent.

Signed-off-by: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>

### CODE REVIEW

**Comment by cyrillos** [comment]
Ack. Thanks!

**Comment by Snorch** [comment]
@cyrillos Thanks for a review! But it looks like I need some more... When preparing a test I've found another related problem in dump_one_eventpoll.

So I've added another fix and a test to differentiate the problem.

File: test/zdtm/static/epoll01.c [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#include <sys/types.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <inttypes.h>
+
+#include "zdtmtst.h"
+
+const char *test_doc	= "Check for epoll";
```
Comment (avagin) [inline test/zdtm/static/epoll01.c:22]: Could you add a comment that explains what this test checks? It can be the description from the commit message.

Comment (Snorch) [reply test/zdtm/static/epoll01.c:22]: I've already put to commit message:
```
This adds three epoll targets on tfd 702 and then adds two epoll targets
on tfd 701. This test is for off calculation in dump_one_eventpoll, the
reverse order makes qsort to actually work.
```
Isn't this enough?

**Review by cyrillos — APPROVED** [review]

File: test/zdtm/static/epoll01.c (hunk 1 above)
Comment (Snorch) [reply test/zdtm/static/epoll01.c]: After rereading your message once again, look like you just ask to add description as a comment in code. Sure - no problem. Done.

**Pull request merged**